modules/extensions-spring-support/src/test/java/org/mule/runtime/module/extension/internal/capability/xml/TestAnotherDocumentedConfig.java

```diff
@@ -14,7 +14,7 @@
 /**
  * This is some Another Config documentation.
  */
-@Configuration(name = "anotherConfig")
+@Configuration(name = "another")
```

Comment:
it also seems to me that part of the problem is that ConfigLoaderDelegate is appending `-config` instead of appending `Config`